components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -528,14 +528,15 @@ export type GitpodService = GitpodServiceImpl<GitpodClient, GitpodServer>;
 const hasWindow = typeof window !== "undefined";
 const phasesOrder: Record<WorkspaceInstancePhase, number> = {
     unknown: 0,
-    preparing: 1,
-    pending: 2,
-    creating: 3,
-    initializing: 4,
-    running: 5,
-    interrupted: 6,
-    stopping: 7,
-    stopped: 8,
+    building: 1,
```

Comment:
Both of those have to swap: preparing is 1.

Reply:
11be9bcaa